.github/workflows/cmake.yml

```diff
@@ -1,6 +1,9 @@
 name: CMake
 
-on: [push]
+on: 
+  push:
+    branches-ignore:
+      - mikado-method-from-video
```

Comment:
Why didn't you put the regexp here?

Reply:
It was not properly saved on my machine. 
I submitted a fix for that